router/src/lib.rs

```diff
@@ -63,3 +63,15 @@ pub type RouterRef = Arc<dyn Router + Sync + Send>;
 pub trait Router {
     async fn route(&self, req: RouteRequest) -> Result<Vec<Route>>;
 }
+
+#[derive(Clone, Debug, Deserialize, Serialize, Default)]
+pub struct RouteCacheConfig {
+    // enable route cache, default false
+    enable: bool,
```

Comment:
I guess the cache should be always enabled.

Reply:
I guess route cache is not necessary, and route cache may return wrong route endpoint. user can enable it if they need.